npm/build-docs.js

```diff
@@ -26,7 +26,7 @@ module.exports = function (exit) {
     }
 
     exec(`${IS_WINDOWS ? '' : 'node'} ${path.join('node_modules', '.bin', 'jsdoc')}${IS_WINDOWS ? '.cmd' : ''}` +
-        ` -c .jsdoc-config.json -u docs lib --query 'pkgVersion=${pkg.version}'`, function (code) {
+        ` -c .jsdoc-config.json -u docs lib --query 'pkgVersion=${pkg.version}' -p`, function (code) {
```

Comment:
What's the `-p` for?

Reply:
Private members are not shown in the generated output unless JSDoc is run with the -p/--private command-line option. This is done to generate documentation for members marked @private as well.

Reply:
I see 😄